internal/operands/common-templates/constants.go

```diff
@@ -11,6 +11,8 @@ const (
 	TemplateWorkloadLabelPrefix  = "workload.template.kubevirt.io/"
 	TemplateDeprecatedAnnotation = "template.kubevirt.io/deprecated"
 
+	TemplateTypeLabelBaseValue = "base"
```

Comment:
can you please update these places https://github.com/kubevirt/ssp-operator/blob/master/internal/operands/common-templates/reconcile.go#L178 and all occurances here https://github.com/kubevirt/ssp-operator/blob/157b424d10a5c81813ce1bbcb66e326c42b0715b/internal/operands/common-templates/reconcile_test.go with this variable?

Reply:
Ok